pkg/docker/util.go

```diff
@@ -16,8 +16,8 @@ import (
 	"github.com/openshift/source-to-image/pkg/util/user"
 )
 
-// DockerImageReference points to a Docker image.
-type DockerImageReference struct {
+// ImageReference points to a Docker image.
+type ImageReference struct {
```

Comment:
This gets used [only once](https://github.com/openshift/source-to-image/pull/492/files#diff-7ae9cffe2e2079019c59cc3416502721L145) as:

```diff
- dockerImageReference, err := docker.ParseDockerImageReference(builder.config.BuilderImage)
+ dockerImageReference, err := docker.ParseImageReference(builder.config.BuilderImage)
```

Sorry not clear if you want me to undo the renaming or was just a rant?